pkg/types/v2/file_existence.go

```diff
@@ -26,11 +27,15 @@ import (
 	"github.com/GoogleContainerTools/container-structure-test/pkg/utils"
 )
 
+var defaultOwnership = -1
+
 type FileExistenceTest struct {
 	Name        string `yaml:"name"`        // name of test
 	Path        string `yaml:"path"`        // file to check existence of
 	ShouldExist bool   `yaml:"shouldExist"` // whether or not the file should exist
 	Permissions string `yaml:"permissions"` // expected Unix permission string of the file, e.g. drwxrwxrwx
+	Uid         int    `yaml:"uid"`         // ID of the owner of the file
```

Comment:
Looks like you are mimicking Adding new optional fields by setting Uid and gid to -1 defaults.